gsoc.yml

```
@@ -1,3 +1,31 @@
+gsoc2019:
+  Sedictious:
+    rss_feed: ""
+    project: 'poliastro'
+  jorgepiloto:
+    rss_feed: ""
+    project: 'poliastro'
+  kakirastern:
+    rss_feed: "https://medium.com/feed/@krisastern"
+    project: 'SunPy'
+  vatch123:
+    rss_feed: "https://medium.com/feed/@vatsalyachaubey19980"
+    project: 'SunPy'
+  vn-ki:
+    rss_feed: "https://medium.com/feed/@appukuttancr"
+    project: 'SunPy'
+  ​yashrsharma44:
+    rss_feed: "https://medium.com/feed/@yashrsharma44"
+    project: 'SunPy'
+  ​tcjansen:
+    rss_feed: "https://astrotiff.home.blog/feed/***" # FIXME add gsoc tag
+    project: 'Astropy'
+  aquatiko:
+    rss_feed: "https://aquatiko.github.io/series/gsoc/"
```

@aquatiko I need the rss address for that series. Would that be: 
```suggestion
    rss_feed: "https://aquatiko.github.io/series/gsoc/index.xml"
```